src/components/AdminDashboard/AdminDashboard.js

```diff
@@ -54,7 +54,9 @@ const AdminDashboard = () => {
 
   // loads incident data on first render
   useEffect(() => {
-    easyMode.fetchIncidents();
+    easyMode.fetchIncidents('approved');
+    easyMode.fetchIncidents('pending');
+    easyMode.fetchIncidents('form-responses');
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Please consider and look into 'why' eslint is barking at your dependency array here. It might be worth a global disable, or fix it so that it satisfies the patterns built into the React useEffect hook paradigm.